protocol/consistencychecks_test.go

```diff
@@ -3,29 +3,28 @@ package protocol
 import "testing"
 
 var (
-	key = []byte("key")
+	alice = "alice"
+	bob   = "bob"
+	key   = []byte("key")
 )
 
-func doRequestAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
-	requestType int, name string) error {
-	switch requestType {
-	case RegistrationType:
-		request := &RegistrationRequest{
-			Username: name,
-			Key:      key,
-		}
-		res, _ := d.Register(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case KeyLookupType:
-		request := &KeyLookupRequest{
-			Username: name,
-		}
-		res, _ := d.KeyLookup(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case MonitoringType:
-	case KeyLookupInEpochType:
+func registerAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
```

Comment:
I guess my point is the purpose of `consistencychecks_test.go` is to test the consistency verification. 
> You're saying that since it's covered there, it's redundant here

What I'm trying to say is that since we already have a test for this there, we can quite confident about the checks here. Maybe my sense was wrong :(